### PR TITLE
Fixed for new staff team

### PR DESCRIPTION
**Pull request by dwilliamsuk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -104,12 +98,6 @@ <h2 class="subtitle">Who's in charge?</h2>
         </div>
 
         <div class="column">
-          <a href="https://auttaja.us" target="_blank" rel="noopener noreferrer"><img src="/img/people/kelly.gif" class="someone mod"></a>
-          <h3>Kelwing</h3>
-          <span class="tag is-warning">Moderator</span>
-        </div> 
-
-        <div class="column">
```
Comment (t5r7) [inline index.html:112]: looks like you deleted the first bit of vlad's column

Comment (dwilliamsuk) [reply index.html:112]: o u fixe fixe 4 me plz?

Comment (t5r7) [reply index.html:112]: don't think i can mate

File: index.html [hunk 2]
```diff
@@ -65,20 +65,14 @@ <h2 class="subtitle">Who's in charge?</h2>
...
           <img src="/img/people/ren.png" class="someone admin">
           <h3>Renekton</h3>
           <span class="tag is-info">Admin</span>
         </div>
 
         <div class="column">
           <a href="https://jaffasite.ga" target="_blank" rel="noopener noreferrer"><img src="/img/people/jaffa.png" class="someone admin"></a>
-          <h3><span title="AKA TheJaffaMeme">TheJaffaMan</span></h3>
+          <h3><span title="TheJaffaMeme">TheJaffaMeme</span></h3>
```
Comment (ghost) [inline index.html]: Please remove the `title=" ... "` bit if its contents are the exact same as the text shown

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html:112]: this is fine, it's just GitHub's diff being weird

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**